File: 프로그래머스/1/12954. x만큼 간격이 있는 n개의 숫자/x만큼 간격이 있는 n개의 숫자.java
import java.util.*;
import java.math.*;
class Solution {
    public double[] solution(int x, int n) {
        double[] answer = new double[n];
        
        for(int i=1;i<=n;i++)
        {
            if(x ==0)
            {
                answer[i-1] = 0;
            }
            else
            {
                answer[i-1] = x * (double)i;
            }
        }
        
        
        return answer;
    }
}

File: 프로그래머스/1/12954. x만큼 간격이 있는 n개의 숫자/x만큼 간격이 있는 n개의 숫자.js
function solution(x, n) {
    var answer = [];
    
    for(let i =0;i<n;i++)
        {
            answer.push((i+1)*x);
        }
    
    const arr = Array(n).fill(x).map((value,index) => value*(index+1));
    console.log(arr);
    return answer;
}